packages/capnpc-ts/src/file.ts

```diff
@@ -98,7 +98,12 @@ export function getJsType(ctx: CodeGeneratorFileContext, type: s.Type, construct
 
     case s.Type.INTERFACE:
 
-      return 'capnp.Interface';
+      const i = getFullClassName(lookupNode(ctx, type.getInterface().getTypeId()));
+
+      // TODO: is this snippet relevant?
```

Comment:
This is fine, but will be overwritten soon. The compiler is getting some nice upgrades.